src/Http/Client/Message.php

```diff
@@ -166,20 +159,4 @@ public function cookies(): array
     {
         return $this->_cookies;
     }
-
-    /**
-     * Get/set the body for the message.
-     *
-     * @param string|null $body The body for the request. Leave null for get
-     * @return mixed Either $this or the body value.
-     */
-    public function body($body = null)
```

Comment:
This wasn't deprecated before. We should document this removal in the 4.0 migration guide.

Reply:
Done